eureka2-core/src/main/java/com/netflix/eureka2/health/AbstractHealthStatusProvider.java

```diff
@@ -34,6 +34,4 @@ public boolean moveHealthTo(Status newStatus) {
         healthSubject.onNext(new HealthStatusUpdate<SUBSYSTEM>(newStatus, descriptor));
         return true;
     }
-
-    protected abstract Status toEurekaStatus(Status status);
```

Comment:
Syntactically not, but conceptually yes, so I would keep it that way. We may also reintroduce some abstract methods in the future.